app/src/main/kotlin/com/jakewharton/sa4p/ui/MainUi.kt

```diff
@@ -13,10 +13,14 @@ import androidx.compose.ui.Modifier
 import androidx.compose.ui.res.stringResource
 import com.jakewharton.sa4p.R
 import com.jakewharton.sa4p.presenter.MainModel
+import kotlinx.collections.immutable.toPersistentList
 
 @OptIn(ExperimentalMaterial3Api::class)
 @Composable
-fun MainUi(model: MainModel) {
+fun MainUi(
+	model: MainModel,
+	modifier: Modifier = Modifier,
```

Comment:
This is unused. Either we need to hook it up correctly to the scaffold or suppress the lint check.